core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -330,8 +330,8 @@ public FolderId findFolder(FolderId baseFolderId, String folderName) throws File
 		if (StringUtils.isEmpty(folderName)) {
 			return baseFolderId;
 		}
-		ExchangeFolderReference targetFolder = new ExchangeFolderReference(folderName, getMailAddress(), baseFolderId, getMailboxFolderSeparator());
-		ExchangeService exchangeService = getConnection();
+		ExchangeObjectReference targetFolder = asObjectReference(folderName);
```

Comment:
Hier raakt het baseFolderId kwijt, en daarom wordt in mijn test setup de basefolder niet gevonden als hij een subfolder is van Inbox.